### PR TITLE
Adding documentation for WriteBatch size limit 

### PR DESCRIPTION
**Pull request by thebrianchen**
Reference: b/70222712

### CODE REVIEW

File: Firestore/Source/Public/FIRFirestore.h [hunk 1]
```diff
@@ -110,6 +110,10 @@ NS_SWIFT_NAME(Firestore)
  * Executes the given updateBlock and then attempts to commit the changes applied within an atomic
  * transaction.
  *
+ * The maximum number of writes allowed in a single transaction is 500, but note that each usage of
+ * `FieldValue.serverTimestamp() `FieldValue.arrayUnion()`, `FieldValue.arrayRemove()`, or
```
Comment (mikelehen) [inline Firestore/Source/Public/FIRFirestore.h]: missing a comma

File: Firestore/Source/Public/FIRFirestore.h [hunk 2]
```diff
@@ -141,6 +145,10 @@ NS_SWIFT_NAME(Firestore)
  * Creates a write batch, used for performing multiple writes as a single
  * atomic operation.
  *
+ * The maximum number of writes allowed in a single batch is 500, but note that each usage of
+ * `FieldValue.serverTimestamp() `FieldValue.arrayUnion()`, `FieldValue.arrayRemove()`, or
```
Comment (mikelehen) [inline Firestore/Source/Public/FIRFirestore.h]: missing a comma

File: Firestore/Source/Public/FIRFirestore.h [hunk 3]
```diff
@@ -141,6 +145,10 @@ NS_SWIFT_NAME(Firestore)
...
+ * `FieldValue.increment()` inside a transaction counts as an additional write.
```
Comment (mikelehen) [inline Firestore/Source/Public/FIRFirestore.h]: "transaction" => "batch"

File: Firestore/Source/Public/FIRFirestore.h (hunk 1 above)
Comment (wilhuff) [reply Firestore/Source/Public/FIRFirestore.h]: Also, unbalanced backticks, here and throughout.

**Comment by thebrianchen** [comment]
Apologies for not checking my work more clearly.

**Pull request merged**